string[] meshNames
---
int32[] errcodes
string[] errmsgs
